pkg/utils/utils_test.go

```diff
@@ -51,12 +51,12 @@ func TestBuildCSPDirectives(t *testing.T) {
 			contentSecurityPolicy: "",
 			indexPageScriptNonce:  "foobar",
 			want: []string{
-				"base-uri 'self'",
-				"default-src 'self'",
-				"img-src 'self' data:",
-				"font-src 'self' data:",
-				"script-src 'self' 'unsafe-eval' 'nonce-foobar'",
-				"style-src 'self' 'unsafe-inline'",
+				"base-uri 'self' console.redhat.com api.segment.io",
+				"default-src 'self' console.redhat.com api.segment.io",
+				"img-src 'self' console.redhat.com api.segment.io data:",
+				"font-src 'self' console.redhat.com api.segment.io data:",
+				"script-src 'self' console.redhat.com api.segment.io 'unsafe-eval' 'nonce-foobar'",
+				"style-src 'self' console.redhat.com api.segment.io 'unsafe-inline'",
```

Comment:
Do we need to add these domains to each fetch directive? We should limit it to what we really need.